src/components/Reports/ReportEdit.tsx

```diff
@@ -53,6 +50,16 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     } else {
       snackbar.toastError(strings.GENERIC_ERROR, strings.REPORT_COULD_NOT_OPEN);
     }
+  }, [reportIdInt, snackbar]);
+
+  const [currentUserEditing, setCurrentUserEditing] = useState(true);
+  useEffect(() => {
```

Comment:
Moved the `setInterval` into its own `useEffect` and also don't allow its handler to write to the report state. This was changing the state out from under the component and invisibly reverting edits 😱

Reply:
good catch!